#! /bin/sh /usr/share/dpatch/dpatch-run
## default-network.dpatch by Guido Trotter <ultrotter@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use dummy script rather than bridge one as default for xend

@DPATCH@
diff -urNad xen-3.0-3.0.2+hg9583/tools/examples/xend-config.sxp /tmp/dpep.QATq8l/xen-3.0-3.0.2+hg9583/tools/examples/xend-config.sxp
--- xen-3.0-3.0.2+hg9583/tools/examples/xend-config.sxp	2006-04-06 14:17:55.256393320 +0200
+++ /tmp/dpep.QATq8l/xen-3.0-3.0.2+hg9583/tools/examples/xend-config.sxp	2006-04-06 14:18:09.000000000 +0200
@@ -86,7 +86,7 @@
 # two fake interfaces per guest domain.  To do things like this, write
 # yourself a wrapper script, and call network-bridge from it, as appropriate.
 #
-(network-script network-bridge)
+(network-script network-dummy)
 
 # The script used to control virtual interfaces.  This can be overridden on a
 # per-vif basis when creating a domain or a configuring a new vif.  The
